jvb/src/main/java/org/jitsi/videobridge/Main.java

```diff
@@ -92,8 +95,27 @@ public static void main(String[] args)
         ClientConnectionImpl clientConnectionImpl = ClientConnectionSupplierKt.singleton.get();
         clientConnectionImpl.start();
 
+        JvbLoadManager<RtpPacketDelayMeasurement> jvbLoadManager = new JvbLoadManager<>(
+            JvbLoadManagerKt.getRtpPacketDelayThreshold(),
+            new LastNReducer(VideobridgeSupplierKt.singleton.get(), .75)
+        );
+
         Logger logger = new LoggerImpl("org.jitsi.videobridge.Main");
 
+        ScheduledFuture<?> loadManagerTask = TaskPools.SCHEDULED_POOL.scheduleAtFixedRate(
+            () -> {
+                double delay = Endpoint.rtpPacketDelayStats.getSnapshot().getAverageDelayMs();
```

Comment:
Is this the average delay since the bridge came up? We need some instantaneoous measurement

Reply:
I thought this was doing windowed, but it looks like it isn't.  But it sounds like we won't be using this for the initial load measurement anyway (either packet rate or system load).